### PR TITLE
Add additional tag to `next` image with commit sha

### PR DESCRIPTION
**Pull request by dvjn**
Next image will have an additional tag of fomat `next-${truncated_commit_sha}`, where truncated_commit_sha is the first 10 characters of the commit sha (also, same for for the `next-alpine` image).

### CODE REVIEW

**Comment by dvjn** [comment]
@6543 :)

**Comment by 6543** [comment]
`yaml: line 76: did not find expected ',' or ']'`

File: .woodpecker/docker.yml [hunk 1]
```diff
@@ -74,7 +74,7 @@ pipeline:
       repo: woodpeckerci/woodpecker-server
       dockerfile: docker/Dockerfile.server.multiarch
       platforms: *platforms_server
-      tag: next
+      tag: [next, next-${CI_COMMIT_SHA:0:10}]
```
Comment (6543) [inline .woodpecker/docker.yml]:
```suggestion
      tags: next, next-${CI_COMMIT_SHA:0:10}
```

File: .woodpecker/docker.yml [hunk 2]
```diff
@@ -87,7 +87,7 @@ pipeline:
       repo: woodpeckerci/woodpecker-server
       dockerfile: docker/Dockerfile.server.alpine.multiarch
       platforms: *platforms_alpine
-      tag: next-alpine
+      tag: [next-alpine, next-${CI_COMMIT_SHA:0:10}-alpine]
```
Comment (6543) [inline .woodpecker/docker.yml]:
```suggestion
      tags: next-alpine, next-${CI_COMMIT_SHA:0:10}-alpine
```

File: .woodpecker/docker.yml [hunk 3]
```diff
@@ -168,7 +168,7 @@ pipeline:
       repo: woodpeckerci/woodpecker-agent
       dockerfile: docker/Dockerfile.agent.multiarch
       platforms: *platforms_release
-      tag: next
+      tag: [next, next-${CI_COMMIT_SHA:0:10}]
```
Comment (6543) [inline .woodpecker/docker.yml]:
```suggestion
      tags: next, next-${CI_COMMIT_SHA:0:10}
```

File: .woodpecker/docker.yml (hunk 1 above)
Comment (dvjn) [reply .woodpecker/docker.yml]: actually similar syntax is used on line 130, 143, etc.
does this still need updating after adding quotes?

Comment (6543) [reply .woodpecker/docker.yml]: no that also should do it ...

**Review by 6543 — APPROVED** [review]

**Pull request merged**